src/api/test-controller/index.js

```diff
@@ -300,6 +306,19 @@ export default class TestController {
     _expect$ (actual) {
         const callsite = getCallsiteForMethod('expect');
 
+        const addAssertionWarning = (value, builderClassName, warningMessage, warningCallsite) => {
+            const renderedCallsite = renderCallsiteSync(warningCallsite, {
+                renderer:    renderers.noColor,
+                stackFilter: createStackFilter(Error.stackTraceLimit)
+            });
+
+            if (typeof value === 'function' && value[Object.getOwnPropertySymbols(value)[0]] && value[Object.getOwnPropertySymbols(value)[0]].constructor.name === builderClassName)
```

Comment:
You can import `functionBuilderSymbol` directly from 'src/client-functions/builder-symbol.js`